src/starbase/setup.ts

```diff
@@ -92,12 +92,8 @@ async function setupIntegration(integration: StarbaseIntegration) {
  * `main` branch
  */
 async function updateIntegrationDirectory(directory: string) {
-  let repo = await Repository.open(directory);
-  await repo.fetchAll();
-
-  const localMain: Reference = await repo.getCurrentBranch();
-  const originMain = await repo.getBranch('origin/main');
-  await repo.mergeBranches(localMain, originMain);
+  const git = simpleGit(directory);
```

Comment:
If anything, I like this better since we don't have to specify the branch.  Makes life easier for future updates where we allow branches other than main to be pulled.  💯

Reply:
@ceelias Nice. Much simpler interface. Can you validate that still works as expected in Docker?